vita3k/net/src/epoll.cpp

```diff
@@ -1,8 +1,7 @@
 #include <net/epoll.h>
 
 int Epoll::add(int id, abs_socket sock, SceNetEpollEvent *ev) {
-    auto it = eventEntries.find(id);
-    if (it != eventEntries.end()) {
+    if (eventEntries.contains(id)) {
```

Comment:
Replace that function by:
`if(!eventEntries.try_emplace(id, EpollSocket{ ev->events, ev->data, sock }).second)
    return SCE_NET_ERROR_EEXIST;
return 0; `

Reply:
Nice one 👍